Task3/Task3_Atanas_Kisyov/src/python/main.py

```diff
@@ -0,0 +1,118 @@
+import logging
+
+import argparse
+import psycopg2
+
+
+AVAILABLE_TABLES = (
+    'car_washes',
+    'places',
+    'car_wash_places',
+    'services',
+    'car_wash_services',
+    'users',
+    'accounts',
+    'orders',
+    'order_details',
+)
+INSTRUCTION_MESSAGE = 'This is a list of all available tables in the database. ' \
+                      'Please make an input what you want to see.'
+
+
+def add_arguments():
+    """Initialize available arguments for user to pass to the program"""
+
+    parser = argparse.ArgumentParser(prog='Database Connection')
+    parser.add_argument('Mode', help='r=Read Mode, rw=Read/Write')
+    args = vars(parser.parse_args())
+    return args
+
+
+class Singleton:
```

Comment:
Please check metaclass singleton: https://stackoverflow.com/questions/6760685/creating-a-singleton-in-python